db/models/round.py

```diff
@@ -50,10 +50,12 @@ class Round(BaseModel):
     contact_textphone = Column("contact_textphone", db.String(), nullable=True, unique=False)
     support_times = Column("support_times", db.String(), nullable=False, unique=False)
     support_days = Column("support_days", db.String(), nullable=False, unique=False)
-    instructions = Column("instructions", db.String(), nullable=False, unique=False)
+    instructions_json = Column("instructions_json", JSON(none_as_null=True), nullable=True, unique=False)
     feedback_link = Column("feedback_link", db.String(), unique=False)
     project_name_field_id = Column("project_name_field_id", db.String(), unique=False, nullable=False)
-    application_guidance = Column("application_guidance", db.String(), nullable=True, unique=False)
```

Comment:
Are there other PRs coming for the consumer services to deal with reading this from the json?

Reply:
Sorry, I was thinking that services that called the API would now get back JSON instead of just the text value. But I've now looked again and you've sorted that out here anyway, so ignore this :) https://github.com/communitiesuk/funding-service-design-fund-store/pull/276/files#diff-e8bf8155ecc27651ffc56921d9f536dc9717374e8315f9f1cf384ac08513f621R44